content/Watsonx/CodeAssistant/101.md

```diff
@@ -0,0 +1,78 @@
+---
+title: '101: Ansible Lightspeed Environment Setup'
+timeToComplete: 15
+updated: 2023-09-13
+---
+
+# 101: Ansible Lightspeed environment setup in VS Code
+
+## Goal
+
+The goal of this lab is to install the Ansible Lightspeed extension and enable and authenticate the extension in VS Code. 
+
+## Install Ansible extension in VS Code
+
+1. Open the VS Code application.
+2. Click the **Extensions** icon on the left sidebar.
+
+    ![](./images/settings-icon.png)
+3. Search for **Ansible** in the search box and then click the **Install** button.
+
+    ![](./images/search-for-ansible.png)
+
+## Enable the Ansible Lightspeed extension
+
+1. After installing the Ansible extension, click on the little gear icon and select the **Extension Setting** option.
```

Comment:
I had to restart my VS Code before the full set of options showed up.  Initially I got this:
<img width="899" alt="Screen Shot 2023-09-17 at 2 28 31 PM" src="https://github.com/ibm-build-lab/VAD-VAR-Workshop/assets/7364498/0f4ab605-a186-423b-a8fa-3ddd1260d48a">
After restart I see what is in the screenshot.  Can we add a "You may need to restart VS Code before the full set of ansible options is available"?